cs/src/core/Index/FasterLog/FasterLogRecoveryInfo.cs

```diff
@@ -66,7 +69,7 @@ public void Initialize(BinaryReader reader)
                 checkSum = reader.ReadInt64();
                 BeginAddress = reader.ReadInt64();
                 UntilAddress = reader.ReadInt64();
-                if (version == 1)
```

Comment:
nit: why change to > 0? I feel like with version numbers we want to be as future-proof as possible.

Reply:
the goal was that if we updated the version to say 2, the rest of the code would work correctly out of the box as baseline. then we would just make the changes necessary for whatever caused us to move to version 2.